SQL/0000-00-00-schema.sql

```diff
@@ -725,6 +727,7 @@ CREATE TABLE `notification_types` (
   `Type` varchar(255) NOT NULL default '',
   `private` tinyint(1) default '0',
   `Description` text,
+  `Origin` varchar(255) DEFAULT NULL,
```

Comment:
Origin should be on the notification_spool table, not the notification_types table.